duckduckgo/macos-browser
# Update DDG home icon

Task/Issue URL: https://app.asana.com/0/1177771139624306/1202444402883584/f

**Description**:
Replace DDG favicon 
Before
<img width="618" alt="Before" src="https://user-images.githubusercontent.com/782316/176157147-5eb34d0e-3d1b-4e0d-a456-07464a2c1bc9.png">

After
<img width="772" alt="After" src="https://user-images.githubusercontent.com/782316/176157137-8139891e-87a6-4478-96a6-2729cea82e9f.png">

**Steps to test this PR**:
1. Open a new tab and check if the tab icon for DDG looks good
2. Check if the asset from the asana task is the one being used
